source/developer/localization.rst

```diff
@@ -130,7 +129,7 @@ If you're interested in contributing to the process, please join the `Mattermost
     "Türkçe - Turkish", "`Kaya Zeren <https://twitter.com/kaya_zeren>`__", ""
     "Pусский - Russian", "`Alexey Napalkov <https://github.com/flynbit>`_", ""
     "Yкраїнська - Ukrainian", "TBD (Open Role)", ""
-    "한국어 - Korean", "TBD (Open Role)", ""
+    "한국어 - Korean", "`avasconcelos114 <https://translate.mattermost.com/user/avasconcelos114/>`__", ""
```

Comment:
Confirming Andre agreed to be the official reviewer/translator for Korean?

Reply:
Actually no, you are right. Removed.